### PR TITLE
[eos] Update readme and .rst files

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### SUMMARY
- updated README.md
- add .rst files for all modules

##### ISSUE TYPE

- Feature Pull Request

### CODE REVIEW

**Comment by trishnaguha** [comment]
@GomathiselviS Can we merge this PR?

File: plugins/modules/eos_ospfv2.py [hunk 1]
```diff
@@ -549,6 +550,59 @@
...
+          count: 8000
+          ignore_count: 3
+          ignore_time: 6
+          reset_time: 20
+          threshold: 40
+      - process_id: 3
+        vrf: vrf02
+        redistribute:
+        - routes: connected
+>>>>>>> rerun
```
Comment (Qalthos) [inline plugins/modules/eos_ospfv2.py:605]: This looks like a missed merge conflict?

**Comment by GomathiselviS** [comment]
recheck

**Comment by GomathiselviS** [comment]
recheck

File: plugins/modules/eos_ospfv2.py [hunk 2]
```diff
@@ -549,6 +550,58 @@
...
+        max_lsa:
+          count: 8000
+          ignore_count: 3
+          ignore_time: 6
+          reset_time: 20
+          threshold: 40
+      - process_id: 3
+        vrf: vrf02
+        redistribute:
+        - routes: connected
```
Comment (Qalthos) [inline plugins/modules/eos_ospfv2.py:604]: I'm almost positive you didn't actually want any of this. It's nearly identical to the example above it, & is at the wrong indentation

```suggestion
```

File: README.md [hunk 3]
```diff
@@ -7,11 +7,65 @@ The Ansible Arista EOS collection includes a variety of Ansible content to help
 
 This collection has been tested against Arista EOS 4.20.10M.
 
+<!--start requires_ansible-->
+## Ansible version compatibility
+
+This collection has been tested against following Ansible versions: **>=2.9**.
```
Comment (samccann) [inline README.md:13]: I've seen others that have this as >2=2.9<2.11.

**Comment by GomathiselviS** [comment]
recheck

**Pull request merged**